shacl12-test-suite/tests/sparql/node/sparql-001.ttl

```diff
@@ -28,11 +28,11 @@ ex:TestShape
 ex:TestShape-sparql
   sh:message "Cannot have a label" ;
   sh:prefixes <http://datashapes.org/sh/tests/sparql/node/sparql-001.test> ;
-  sh:select """
```

Comment:
This hunk of lines is also affected by end of line changes.

I just noticed that the previous files with EOL changes were from the SHACL 1.0 test suite.  Is this PR's scope 1.2's tests, or going back in time to 1.0 too?